rows.go

```diff
@@ -142,6 +150,25 @@ func parseDateTimeValue(columnType string, value string) (driver.Value, error) {
 	return nil, fmt.Errorf("type not known: %s", columnType)
 }
 
+func parseFloatValue(val interface{}) (float64, error) {
+	fmt.Printf("parseFloatValue: %v\n", val)
+	if _, ok := val.(string); ok {
+		switch val.(string) {
+		case "inf":
```

Comment:
How does this work with numbers? when val is "0.123" cast to string returns an error? Should we change `ok` to something like `not_num` for clarity?

Reply:
When the value is an actual float, we receive a json float value. So this one is only for cases when we receive a string for a float type